### PR TITLE
Fixing token names for change to CR

### LINKED ISSUE #126: Crimson Vow CR changes the names of most tokens in the game.
The article detailing the changes is [here](https://magic.wizards.com/en/articles/archive/news/comprehensive-rules-changes-2021-11-10). In short, all generic tokens (Blood, Human Soldier, Thopter, etc) will now have the word Token added to their name. So instead of Blood it is now Blood Token, Human Soldier Token, Thopter token, etc. Any tokens that are created with a specific name (like [Jiang Yanggu creating Mowu](https://scryfall.com/card/gs1/22/jiang-yanggu)) are unaffected.

### PR DESCRIPTION
**Pull request by mainman879**
Fixes #126

### CODE REVIEW

**Comment by tooomm** [comment]
FYI: If you put the phrase "Fixes #126" at the beginning of the PR description, they will be connected to each other and the linked issue will be automatically closed once this PR gets merged.

Thanks again for your recent works in keeping things updated around here!

**Comment by mainman879** [comment]
> FYI: If you put the phrase "Fixes #126" at the beginning of the PR description, they will be connected to each other and the linked issue will be automatically closed once this PR gets merged.
> 
> Thanks again for your recent works in keeping things updated around here!

Thanks for letting me know. I updated the description but idk if it works on edits, I'm still pretty clueless to githubs intricacies. And its no problem keeping the tokens file up to date, I benefit from it being up to date just as much as anyone else.

File: tokens.xml [hunk 1]
```diff
@@ -517,7 +517,6 @@ Creature tokens you control have flying and vigilance.</text>
             <set picURL="https://img.scryfall.com/cards/large/front/3/f/3fc3a29a-280d-4f2c-9a01-8cfead75f583.jpg">EVE</set>
             <set picURL="https://img.scryfall.com/cards/large/front/c/3/c3cbd01b-dbf5-424a-855d-f9a9d7e7e414.jpg">LRW</set>
             <set picURL="https://img.scryfall.com/cards/large/front/d/0/d02d5670-fff3-4011-8e45-e6843e4e8988.jpg">DST</set>
-            <reverse-related>"Beast Without"</reverse-related>
```
Comment (ebbit1q) [inline tokens.xml:520]: is this change significant?

**Comment by tooomm** [comment]
Looks like you came up with a superseding PR in #129 that addresses all cards? @ebbit1q

**Comment by ebbit1q** [comment]
yes, I automated it, mainmans work still helped while making that though

File: tokens.xml (hunk 1 above)
Comment (mainman879) [reply tokens.xml:520]: The "Beast Without" entry should be removed. It was a leftover from scryfalls spoiler name for Sawtusk Demolisher.

**Comment by mainman879** [comment]
Handled by https://github.com/Cockatrice/Magic-Token/pull/129

**Pull request closed without merging**